ivy/functional/frontends/numpy/fft/discrete_fourier_transform.py

```diff
@@ -125,3 +125,18 @@ def rfftfreq(n, d=1.0):
     N = n // 2 + 1
     results = ivy.arange(0, N, dtype=int)
     return results * val
+
+
+@to_ivy_arrays_and_back
+@with_supported_dtypes
```

Comment:
[Here](https://unify.ai/docs/ivy/overview/deep_dive/data_types.html#supported-and-unsupported-data-types) is the documentation on how to use this decorator correctly.